README.md

```diff
@@ -1,3 +1,23 @@
-# InterviewFlashCards
+# Interview Flash Cards
+An open-source project designed to be a tool for iOS developers preparing for an interview. The project consists of data structure and algorithm questions, as well as iOS technical questions, all in flashcard form.
 
- Collection of iOS technical and data structures and algorithms questions in flashcard form. 
+## Pods
```

Comment:
we can do without this section as the Podfile list will most likely change over time